dunstctl

```diff
@@ -64,15 +72,29 @@ case "${1:-}" in
 	"history-pop")
 		method_call "${DBUS_IFAC_DUNST}.NotificationShow" >/dev/null
 		;;
-	"running")
-		property_get running | ( read -r _ _ paused; printf "%s\n" "${paused}"; )
+	"is-paused")
+                running=$(property_get running | ( read -r _ _ running; printf "%s\n" "${running}"; ))
+                # invert boolean to indiciate pause status rather than running one
+                paused=$(invert_boolean "${running}")
+                printf "%s\n" "${paused}"
 		;;
-	"set-running")
+	"set-paused")
 		[ "${2:-}" ] \
-			|| die "No status parameter specified. Please give either 'true' or 'false' as running parameter."
-		[ "${2}" = "true" ] || [ "${2}" = "false" ] \
-			|| die "Please give either 'true' or 'false' as running parameter."
-		property_set running variant:boolean:"${2}"
+			|| die "No status parameter specified. Please give either 'true', 'false' or 'toggle' as paused parameter."
+		[ "${2}" = "true" ] || [ "${2}" = "false" ] || [ "${2}" = "toggle" ] \
+			|| die "Please give either 'true', 'false' or 'toggle' as paused parameter."
+                if [ "${2}" = "toggle" ]; then
+                    running=$(property_get running | ( read -r _ _ running; printf "%s\n" "${running}"; ))
+                    if [ "${running}" = "true" ]; then
+                        property_set running variant:boolean:false
+                    else
+                        property_set running variant:boolean:true
+                    fi
+                else
+                    # invert boolean to indiciate running status rather than pause one
+                    running=$(invert_boolean "${2}")
+		    property_set running variant:boolean:"${running}"
```

Comment:
What's up with the indentation here?

Also: dunstctl uses tabs for indentation, not spaces. It looks like you forgot to match your editor settings.

Reply:
Done.